src/consensus/params.cpp

```diff
@@ -218,12 +258,113 @@ namespace Consensus {
         vFundingStreams[idx] = FundingStream::ParseFundingStream(*this, keyConstants, startHeight, endHeight, strAddresses);
     };
 
-    FundingStreamAddress FundingStream::RecipientAddress(const Consensus::Params& params, int nHeight) const
+    void Params::AddZIP207LockboxStream(
+        const KeyConstants& keyConstants,
+        FundingStreamIndex idx,
+        int startHeight,
+        int addressChangeInterval,
+        int intervalCount)
```

Comment:
Why do we have an `addressChangeInterval` when there are no addresses?
```suggestion
        int duration)
```

Reply:
It should perhaps be called "streamChangeInterval" - it's defined as "AddressChangeInterval" in ZIP 207. Funding streams have previously been aligned to change on 1/48 of a halving cycle, for the purpose of recipient address rotation, and this alignment doesn't change with the lockbox proposal.

Reply:
It turns out there was already a `Consensus::Params` member variable that has the value we need here.

Reply:
I don't understand. For the lockbox stream, there are no addresses so the "address change interval" is unobservable.

Reply:
The change interval is really a stream change interval, not an address change interval.